.github/scripts/comment_plan.js

```diff
@@ -37,10 +29,10 @@ Plan: ${ creates } to add, ${ updates } to change, ${ deletes } to destroy
 <summary>Show Plan</summary>
 
 \`\`\`terraform
-${plan}
+${ process.env.PLAN }
```

Comment:
This has to be the plan variable as I do a transform on line 20, I have to add 4 spaces to the beginnings of each line

Reply:
nvm this works...